geant4.sh

```diff
@@ -40,11 +40,30 @@ cmake $SOURCEDIR                                    \
   -DCMAKE_STATIC_LIBRARY_C_FLAGS="-fPIC"            \
   -DGEANT4_USE_G3TOG4=ON                            \
   -DGEANT4_INSTALL_DATA=ON                          \
-  -DGEANT4_USE_SYSTEM_EXPAT=OFF
+  -DGEANT4_USE_SYSTEM_EXPAT=OFF                     \
+  ${XERCESC_ROOT:+-DGEANT4_USE_OPENGL_X11=ON -DGEANT4_USE_GDML=ON -DXERCESC_ROOT_DIR=$XERCESC_ROOT}
```

Comment:
These are necessary options to build G4PY. These options only become active if XercesC has been specified in the `requires:` section, otherwise they are not set. If somebody is using XercesC, then they are most likely trying to build G4PY and know to also set the OpenGL bindings as a requirement. 
Unfortunately I cannot test this recipe on a mac, due to a lack of hardware, but Fons has been using G4PY on his mac and this is directly taken from the makefile he is currently using.